util.go

```diff
@@ -91,7 +92,8 @@ func getPathToBin(target string) (string, error) {
 			id := action.OutputIds[0]
 			for _, artifact := range result.Artifacts {
 				if artifact.ID == id {
-					return artifact.ExecPath, nil
+					binaryPath := strings.TrimPrefix(artifact.ExecPath, "bazel-out/")
```

Comment:
My worry about this is that the return value of `getPathToBin()` is now functionally useless unless you happen to remember to call `bazel info output_path` and join the paths yourself. Given that the type system isn't going to do us any favors (because this is Go), I'd like to avoid that :) 

Two options:

1. Have the `bazelOutputPath` be an additional argument to this function, to make it explicit that the output path is a dependency for this function to produce any useful output. Then the `join()` can just happen inside this function.
2. Just call `bazel info output_path` in this function -- it'll be slower since you'll be executing that once per call to this function, but that might not be a huge deal.

Reply:
Good point, went with (2).